go.mod

```diff
@@ -37,6 +37,8 @@ require (
 	google.golang.org/protobuf v1.28.0
 )
 
+require gopkg.in/yaml.v3 v3.0.1
```

Comment:
Why so special?

Reply:
i guess it's because of removing `// indirect`. why? i do not know, my `go mod tidy` decided to do so

Reply:
Can we manually stuff it into existing `require` block?

Reply:
oh, you meant including that to the common section. got you, moved